dotcom-rendering/src/web/server/document.tsx

```diff
@@ -20,20 +20,29 @@ interface Props {
 }
 
 const generateScriptTags = (
-	scripts: Array<{ src: string; legacy?: boolean }>,
+	scripts: Array<{ src: string; legacy?: boolean } | false>,
 ) =>
-	scripts.reduce((scriptTags, script) => {
-		let attrs = 'defer';
-
-		if (Object.prototype.hasOwnProperty.call(script, 'legacy')) {
-			attrs = script.legacy ? 'defer nomodule' : 'type="module"';
+	scripts.reduce<string[]>((scriptTags, script) => {
+		if (script === false) return scriptTags;
+
+		let attrs: string;
+		switch (script.legacy) {
+			case true:
+				attrs = 'defer nomodules';
```

Comment:
Small typo here? 👩‍🏫 --> `nomodules` instead of `nomodule`

Reply:
```suggestion
				attrs = 'defer nomodule';
```